README.md

```diff
@@ -1 +1,24 @@
 # Slurm rocks
+
+The `slurm-rocks` repository is a collection of files to build rocks for the upstream
+Slurm services.
+
+## Getting started
+
+To get started with the project, you must have [Rockcraft] and [Just] installed for
```

Comment:
[nit]: We could say "To get started with the Slurm rocks" here.

```suggestion
To get started with the Slurm rocks, you must have [Rockcraft] and [Just] installed for
```

Reply:
Fixed